### PR TITLE
feat(wren-ui): Add settings UI & discord 

### PR DESCRIPTION
**Pull request by andreashimin**
## Description

- Add settings UI
- Add Discord link

## Tasks

- [x] Add settings button to sidebar
- [x] Add settings modal
- [x] Add Data Source Settings
    - [x]  Big Query Properties (default)
- [x] Add Cache Settings (revert)
- [x] Add Project Settings
- [x] Add Discord 

## UI Screenshot
- sidebar
image

- Data source settings
image
image

- Cache settings
image
image

- Project settings
image
image

### CODE REVIEW

File: wren-ui/src/utils/error/dictionary.ts [hunk 1]
```diff
@@ -40,4 +40,15 @@ export const ERROR_TEXTS = {
       REQUIRED: 'Please select at least one table.',
     },
   },
+  CACHE: {
```
Comment (fredalai) [inline wren-ui/src/utils/error/dictionary.ts:43]: (NIT) we can remove the until support cache and then add it back.

File: wren-ui/src/components/pages/setup/dataSources/BigQueryProperties.tsx [hunk 2]
```diff
@@ -52,7 +57,10 @@ const UploadCredentials = (props) => {
   );
 };
 
-export default function BigQueryProperties() {
```
Comment (fredalai) [inline wren-ui/src/components/pages/setup/dataSources/BigQueryProperties.tsx:55]: We need to deal with the `DuckDBProperties`, right?

**Review by fredalai — APPROVED** [review]
Beside a question , others LGTM

File: wren-ui/src/components/pages/setup/dataSources/BigQueryProperties.tsx (hunk 2 above)
Comment (andreashimin) [reply wren-ui/src/components/pages/setup/dataSources/BigQueryProperties.tsx:55]: Nope. All DuckDB properties can be updated.

**Pull request merged**